open_seq2seq/encoders/tdnn_encoder.py

```diff
@@ -195,6 +201,8 @@ def _encode(self, input_dict):
               regularizer=regularizer,
               training=training,
               data_format=data_format,
+              drop_block_prob=drop_block_prob,
+              drop_block=(drop_block_index == idx_convnet),
```

Comment:
drop_block_index which by default is -1 indicates that no blocks are ever dropped.
Is this the intended behaviour?

Reply:
drop_block_index corresponds to dropping blocks at inference only. During training the mechanism of dropping blocks is controlled entirely by parameter drop_block_prob.